public/javascripts/Gallery/src/modal/Gallery.Panorama.js

```diff
@@ -0,0 +1,283 @@
+/**
+ * A holder class that inserts a GSV Pano into the supplied DOM element
+ * 
+ * @param {HTMLElement} svHolder The DOM element that the GSV Pano will be placed in
+ * @returns {GalleryPanorama} The gallery panorama that was generated
+ */
+ function GalleryPanorama(svHolder) {
+    var self = {
```

Comment:
I think that this code is mostly copied from elsewhere in our codebase, right? If so, just mention where it's copied from at the top. It might also be worth changing the `var`s to `let`s or whatever is appropriate.

Reply:
There's still a decent amount of audit page stuff in this file. Stuff still needs to be removed, tons of comments need to be updated, variables probably renamed.

Reply:
Seems to be mostly resolved. Again, this'll be in the PR for issue #2582